simulation/test_torrent_status.cpp

```diff
@@ -76,35 +99,82 @@ TORRENT_TEST(status_timers)
 			// to stay within that range
 			if (ticks > 20 * 60 * 60)
 			{
-				// unfortunantly we can't test this many ticks
-				TEST_ERROR("This line should be reached but doesn't");
 				return true;
 			}
 
-			if (ticks == 0){
-				// torrent get ready for seeding on first tick, which means +1s
-				startseed_time = lt::clock_type::now();
+			if(tick_is_in_active_range){
+				// 1 second per tick
+				active_time++;
 			}
 
+			// resume on a tick which is far away from next tick check intervall
+			// because with very low upload limit it takes some time to unchock
+			switch(ticks)
```

Comment:
That could help to keep the 2/3 execution time of the 70000 paused torrent ticks a little bit lower. I could add a comment to the test that we need to test a setup where both sessions have a long tick_interval. But I don't want to priorise it. I did already spend 6 hours with improving the test execution time and another 6 hours to write the test and fix the errors. I would like to spend time for the torrent_peer timer test and for the change to time_point instead of reducing test execution time by 20s